packages/cli/src/lib/setup/setupVendor.ts

```diff
@@ -4,6 +4,26 @@ import fs from 'fs-extra';
 import deepClone from 'deep-clone';
 import { isDeepStrictEqual } from 'util';
 
+/*
```

Comment:
I would add these informations on type level and also allow the property, I can push into the PR  branch if okay for you.